modularinputs/python/github_commits/default/inputs.conf

```diff
@@ -0,0 +1,2 @@
+[github_commits]
+python.version = python3
```

Comment:
I'm actually surprised there aren't more of these types of changes, like for custom commands there is also a `python.version` that we should be setting to `python3` here: https://docs.splunk.com/Documentation/Splunk/9.0.1/Admin/Commandsconf

Reply:
In custom search commands `python.version` is already set to `python3`, hence not present in this PR.